Lib/asyncio/tasks.py

```diff
@@ -950,21 +950,21 @@ def callback():
 def create_eager_task_factory(custom_task_constructor):
     """Create a function suitable for use as a task factory on an event-loop.
 
-	    Example usage:
+            Example usage:
```

Comment:
Please fix the indentation change you (accidentally?) applied to much of this docstring. (Maybe your editor inserted tabs instead of spaces?)